projects/js-packages/image-guide/package.json

```diff
@@ -0,0 +1,40 @@
+{
+	"private": true,
```

Comment:
If you do take Jeremy's suggestion to auto-publish to npm, you'll probably need to not mark it private.